### PR TITLE
Exercise 3: The "Holy Grail" Layout

### PR DESCRIPTION
**Pull request by karlsc**

### CODE REVIEW

File: holy-grail/scss/main.scss [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    order:-1;
+    padding:0.5em;
+    text-align:center;
+}
+.HolyGrail-content {
+    // height: 100%;
+    background-color:#999;
+    padding:0.5em;
+    text-align:justify;
+    flex: 1;
```
Comment (ziad-saab) [inline holy-grail/scss/main.scss]: @karlysun what happens if you remove this line? Can you explain why?

**Comment by karlysun** [comment]
@ziad-saab I suppose that because the `nav` is after the `content`, setting his `order` to -1 is bringing it before the `content`.

**Comment by ziad-saab** [comment]
@karlysun I'm talking about the `flex: 1` line :)

**Comment by karlysun** [comment]
Oh I am not 100% sure, but with or without it, the result wasn't exactly the same. I couldn't say exactly.

**Comment by karlysun** [comment]
@ziad-saab Ok I read about it and understand that `flex` is used for proportions. I changed my 2 sidebars to `flex:1` and set my middle one to `flex:3`, so the middle one is 3 times each side bars. I also got rid of some useless lines in my scss. I hope this is ok.

**Pull request closed without merging**